general/g.region/printwindow.c

```diff
@@ -140,13 +159,14 @@ void print_window(struct Cell_head *window, int print_flag, int flat_flag)
                }
              */
 
-            if (!(print_flag & PRINT_SH)) {
+            if (!(print_flag & PRINT_SH) && format != JSON) {
```

Comment:
Why leave the `datum` and `ellipsoid` out of the json response?

Reply:
Fixed